.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: java
+jdk: oraclejdk11
 sudo: false
 dist: trusty
```

Comment:
DSpace builds aren't happening on Travis anymore are they? This file can probably be removed. But if this should still be using Travis the dist should be updated, at least to `dist: bionic`

Reply:
Yes, I was just realizing that -- I don't think travis used anymore. I'll drop this file. 

I see the current build error from LGTM is `Fatal error compiling: invalid flag: --release` which indicates it might not be using a Java 11 JDK. I'm not familiar yet with how that's configured..

Reply:
We dropped Travis when they stopped being free. So, just verifying that you will need to move `dspace-replicate` to use GitHub Actions for the build process (either in this PR or another). It looks like we forgot to do that back when we moved from Travis to GitHub Actions. Here's that file in the main DSpace/DSpace repo (which might be used as a starting point): https://github.com/DSpace/DSpace/blob/main/.github/workflows/build.yml

Reply:
Thanks. I put a build.yml in place in this PR, approximating the old travis one, and confirmed it worked for the Atmire clone: https://github.com/atmire/dspace-replicate/actions/runs/779465153  

I also added a copy of DSpace's `.lgtm.yml` to tell LGTM to use Java 11.